queue/src/fiskaltrust.Middleware.Localization.QueueFR/RequestCommands/CopyCommand.cs

```diff
@@ -41,7 +45,7 @@ public override async Task<(ReceiptResponse receiptResponse, ftJournalFR journal
                 queueFR.CLastHash = hash;
                 journalFR.ReceiptType = "C";
 
-                var duplicateCount = (await GetCountOfExistingCopiesAsync(request.cbPreviousReceiptReference)) + 1;
+                var duplicateCount = GetCountOfExistingCopies(request.cbPreviousReceiptReference) + 1;
```

Comment:
We could just directly call `_copyPayloadRepository.JournalFRGetCountOfCopies` here since `GetCountOfExistingCopies` is just a oneliner now 🤔